source/connection-string/connection-string-spec.rst

```diff
@@ -180,9 +180,9 @@ The values in connection options MUST be URL decoded by the parser. The values c
 - Strings:
     The value
 - Integer:
-    The value parsed as a integer
+  The value parsed as a integer. If the value is the empty string, then a WARN level logging message must be issued and the key is ignored.
```

Comment:
Can MUST be capitalized in both sentences for clarity?

Reply:
And I suggest also using the phrase "and the key MUST be ignored" for the same reason.

Reply:
I have capitalized MUST and added the phrase you suggested.